code/datums/components/storage/storage.dm

```diff
@@ -733,19 +733,6 @@
 	if(rustle_sound)
 		playsound(A, "rustle", 50, 1, -5)
 
-	if(ishuman(user))

```

Comment:
Are you sure about this?
Standard storage behaviour would probably be fine on pouches (clickdrag to take out of storage, click to open)

Reply:
its how pouches work everywhere else, and is in my experience a lot easier

Reply:
basically all this code does is disabling normal slot behaviour for pockets

Reply:
the code i removed disabled default slot behaviour for pockets, which I have removed here

Reply:
switched to commenting it out per the discord conversation